src/Map/src/LatLng.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map;
+
+use Symfony\UX\Map\Exception\InvalidArgumentException;
+
+/**
+ * Represents a geographical point.
+ *
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final readonly class LatLng
+{
+    public function __construct(
+        public float $latitude,
+        public float $longitude,
+    ) {
+        if ($latitude < -90 || $latitude > 90) {
+            throw new InvalidArgumentException(\sprintf('Latitude must be between -90 and 90 degrees, "%s" given.', $latitude));
+        }
+
+        if ($longitude < -180 || $longitude > 180) {
+            throw new InvalidArgumentException(\sprintf('Longitude must be between -180 and 180 degrees, "%s" given.', $longitude));
+        }
+    }
+
+    /**
+     * @return array{latitude: float, longitude: float}
```

Comment:
"latitude" or "lat" ?

Reply:
`lat` and `lng`!

... PHPStan? :D

Reply:
What about phpstan ? You want to add some phpstan CI checks ?  :)

Reply:
Yes!

PHPStan could have tell me that this returns type was wrong, or types were missing on `private array $markers` for example. 

This can saves us a lot of time and energy IMO